uber/config.py

```diff
@@ -592,6 +592,15 @@ def _is_intstr(s):
         c.TRANSFERABLE_BADGE_TYPES.append(_val)
         _day += timedelta(days=1)
 
+"""
```

Comment:
Ohhh right! I always forget about these rarely-used properties. That's much better than doing this, I'll make that change!